### ISSUE TITLE
Suggestion: Allow backwards compatibility

### ISSUE THREAD

**Issue by skellitor301** [deniszholob/factorio-cheat-sheet#387]
Something I was wondering is what were some of the old ratios for older versions of Factorio, but that data is no longer easily accessible due to it being irrelevant to the current version. But, if I were to want to go back and play older versions with older recipes and ratios I'd have to relearn them or scrounge the forums for several year old posts that sometimes don't exist anymore. It'd be nice for the cheat sheet to have an option to be able to switch between Factorio versions for players who want to try out older versions for nostalgia or curiosity.

**Comment by deniszholob** [comment]
@skellitor301  Does this not satisfy your needs?
https://github.com/deniszholob/factorio-cheat-sheet/releases
This is linked to at the top of the page in the "Overview" section

**Comment by skellitor301** [comment]
I did see that but it only goes back to 0.12 and doesn't cover all the info needed for older versions. I know in the v12 version at least it doesn't have info like energy production, so it's partially incomplete, not to mention v15 is entirely missing cause the google doc is deleted. My initial thought is if you guys would be open to the idea to add this data back to the main site as a kind of dropdown menu to swap between the versions. It would be helpful if this info was more accessible and easier to navigate like the main version is on the site. Understandable if you guys don't want to cause I know that's a bit of an undertaking. But the end result would fix some of those issues mentioned and make the data more accessible and easier to utilize.

**Comment by deniszholob** [comment]
I am the only one really who created and maintains this particular website version after v15, the other stuff that came before that is owned by others. Unfortunately the v15 got deleted, not my choice. and the original cheat sheets where created at v12 on imgr or google slides by other people, nothing before hand exists afaik.
So, I backed up what I could in pdf form. As far as trying to dig data through previous versions of factorio... 
Being the only one maintaining this, unfortunately i cant say i can have this on my roadmap for this project as i have limited time and it needs to be allocated elsewhere. However if the community would like to contribute some commits/PRs i would take a look to approve and merge them in.

TLDR; I have backups of what I could find for older versions before my site. Dont have the time to allocate to build older data into the website, but open to help from community.

**Comment by jxu** [comment]
It's silly to expect a cheatsheet to maintain info on the tiny minority of players who go back to old versions. The only thing I support is base game vs space age mod, since both are current.

**Comment by skellitor301** [comment]
> It's silly to expect a cheatsheet to maintain info on the tiny minority of players who go back to old versions. The only thing I support is base game vs space age mod, since both are current.

It's also silly to assume the suggestion was just for a minority of players that play on those versions. Sometimes it's good for posterity to have this information for research in the history of a game, or for any reason to need to know what the game was like or the meta of the builds were back then. Try to imagine what Minecraft would be like if the Minecraft wiki and posterity maintaining sites like the Omniarchive didn't take archiving and maintaining old data that seriously? Old modpacks would cease to exist, people would be less inclined to play with the changes they don't like, it'd be a lesser game if the history wasn't preserved. Similar case here. Yes, factorio isn't as prevalent in it's history as Minecraft, but that doesn't mean maintaining historical information about the game isn't important to multiple kinds of people, and not just the ones that want to try out the older versions.

I'm also not trying to put more effort on the main maintainer, this is more of a suggestion to bring attention to this gap. I can offer what I can with the forum thread I ran that holds some of this info that was community provided, I mainly organized and maintained the thread. [Tips, Designs, & Tricks](https://forums.factorio.com/viewtopic.php?f=18&t=5553)

**Comment by jxu** [comment]
Ok, then that info can go in a wiki, not a cheatsheet. The Minecraft wiki was never that great anyway. It was missing a lot of info on game mechanics.

**Comment by skellitor301** [comment]
Again, you're missing the point of archival information. This kind of information would best fit in the cheat sheet since that's where most people will go for this kind of information.

This is also not an opinion board, this is an issue thread with a posted suggestion to the maintainers of the cheat sheet. You are not a maintainer of the Factorio cheat sheet, and you said yourself that the only thing you care about is the current version of Factorio and space age. So considering the suggestion has nothing to do with your interests in the game, whether or not you find the inclusion of the information has no meaning here and it's just clogging up a suggestion page with irrelevant arguing. Please move on or provide information relevant to the suggestion that is not a biased and irrelevant opinion.

**Comment by jxu** [comment]
It does matter in that there are not enough resources to maintain a current sheet. If the site maintainer puts resources into this, then it is taking away from resources for the current.

**Comment by skellitor301** [comment]
> It does matter in that there are not enough resources to maintain a current sheet. If the site maintainer puts resources into this, then it is taking away from resources for the current.

Again, you're not a maintainer of this project, so you can't speak on behalf of what the resources are. It's not up to you to argue on behalf of what you do not know. You aren't the host nor a maintainer of the site. You're now starting to change your argument and the issue you have at hand. I've asked you to move on or provide relevant information to the suggestion. This is not an opinion thread, stop treating it as such, if the people behind the project want to reject it, that is on them to decide. Not you. This is not the place to make frivolous arguments over matters that are of no concern to you. If you have into that can be useful to this suggestion that is not your apparent need to make yourself the defacto weaker for this project you are not a part of, then feel free to post. Otherwise, do not speak on behalf of the contributors and the people who maintain the site. They can say if the suggestion is worth their time or not, not you
